### PR TITLE
i18n(es): update editor-setup

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Update `editor-setup.md` #1235

### CODE REVIEW

File: src/pages/es/editor-setup.md [hunk 1]
```diff
@@ -27,15 +27,23 @@ Para empezar, instala la [extensión de Astro para VS Code](https://marketplace.
...
 
-- [Extensión de VS Code para Open VSX](https://open-vsx.org/extension/astro-build/astro-vscode)<span style="margin: 0.25em;"><Badge variant="accent">Official</Badge></span> - La extensión official de VS Code, está disponible en el registro the Open VSX para otras plataformas como [VSCodium](https://vscodium.com/).
-- [Extensión de Nova](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/)<span style="margin: 0.25em;"><Badge variant="neutral">Community</Badge></span> - Resaltado de sintaxis, IntelliSense y autocompletado para Astro.
+- [Extensión de VS Code para Open VSX](https://open-vsx.org/extension/astro-build/astro-vscode)<span style="margin: 0.25em;"><Badge variant="accent">oficial</Badge></span> - La extensión official de VS Code, está disponible en el registro the Open VSX para otras plataformas como [VSCodium](https://vscodium.com/).
+- [Extensión de Nova](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/)<span style="margin: 0.25em;"><Badge variant="neutral">Comunidad</Badge></span> - Provee resaltado de sintaxis, IntelliSense y autocompletado para Astro en Nova.
```
Comment (alfredogonzalezmartinez) [inline src/pages/es/editor-setup.md:31]:
```suggestion
- [Extensión de Nova](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/)<span style="margin: 0.25em;"><Badge variant="neutral">Comunidad</Badge></span> - Provee resaltado de sintaxis y autocompletado para Astro en Nova.
```

Comment (alfredogonzalezmartinez) [reply src/pages/es/editor-setup.md:31]: Using "IntelliSense" and "autocompletado" at the same time is redundant.

**Review by alfredogonzalezmartinez — APPROVED** [review]
Nice! A litle change and LGTM

**Comment by kevinzunigacuellar** [comment]
good to go ✅

**Comment by Yan-Thomas** [comment]
Good job @kevinzunigacuellar, @alfredogonzalezmartinez! :purple_heart:

**Pull request merged**